code/modules/clothing/clothing.dm

```diff
@@ -390,12 +390,16 @@
 			if(flags_cover & PEPPERPROOF)
 				things_blocked += "pepperspray"
 			if(length(things_blocked))
-				readout += "\n<b>COVERAGE</b>"
-				readout += "\nIt will block [english_list(things_blocked)]."
+				readout += "<b><u>COVERAGE</u></b>"
+				readout += "It will block [english_list(things_blocked)]."
 
 		readout += "</span>"
```

Comment:
```suggestion
```

you removed the span class notice up there so we should also remove the span escape

Reply:
Tru I even meant to and forgot